### PR TITLE
feat:add i18n for layout

### PR DESCRIPTION
**Pull request by JimmFly**
fix https://github.com/toeverything/AFFiNE/issues/327

动画

### CODE REVIEW

File: libs/components/layout/src/settings-sidebar/Settings/use-settings.ts [hunk 1]
```diff
@@ -97,6 +97,13 @@ export const useSettings = (): SettingItem[] => {
                 message.success('Page link copied successfully');
             },
         },
+        {
+            type: 'button',
+            name: 'Language',
+            onClick: () => {
+                console.log('Language is change');
```
Comment (lawvs) [inline libs/components/layout/src/settings-sidebar/Settings/use-settings.ts:104]:
```suggestion
                // Do nothing
```

File: libs/components/layout/src/i18n/resources/en.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        "Paper": "Paper",
+        "Edgeless": "Edgeless",
+        "Sync to Disk": "Sync to Disk",
+        "Share": "Share",
+        "warningTips": {
+            "isNotfsApiSupported": "Welcome to the AFFiNE demo. To begin saving changes you can SYNC DATA TO DISK with the latest version of Chromium based browser like Chrome/Edge",
+            "isNotLocalWorkspace": "Welcome to the AFFiNE demo. To begin saving changes you can SYNC TO DISK.",
+            "DoNotStore": "AFFiNE is under active development and the current version is UNSTABLE. Please DO NOT store information or data"
+        },
+        "stetting": {
```
Comment (lawvs) [inline libs/components/layout/src/i18n/resources/en.json:12]: typo

```suggestion
        "setting": {
```

File: libs/components/layout/src/i18n/resources/zh.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        "Paper": "页面",
+        "Edgeless": "无边缘",
+        "Sync to Disk": "同步到磁盘",
+        "Share": "分享",
+        "warningTips": {
+            "isNotfsApiSupported": "欢迎来到AFFiNE 的演示界面。您可以使用最新版本的基于Chrome的浏览器（如Chrome/Edge）将数据同步到磁盘来进行保存",
+            "isNotLocalWorkspace": "欢迎来到AFFiNE 的演示界面，您可以同步到磁盘来进行保存操作。",
+            "DoNotStore": "AFFINE 正在积极开发中，当前版本不稳定。请不要存储信息或数据。"
+        },
+        "stetting": {
```
Comment (lawvs) [inline libs/components/layout/src/i18n/resources/zh.json:12]: typo

File: libs/components/layout/src/i18n/index.ts [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+//     <Option key={option.value} value={option.value}>
+//         {option.text}
+//     </Option>
+// ))}
+// </Select>
+
+// const { t, i18n } = useTranslation();
+// const changeLanguage = (event: any) => {
+//     i18n.changeLanguage(event);
+// };
```
Comment (lawvs) [inline libs/components/layout/src/i18n/index.ts:40]:
```suggestion
```

File: libs/components/layout/src/settings-sidebar/ContainerTabs/ContainerTabs.tsx [hunk 5]
```diff
@@ -67,15 +68,15 @@ export const ContainerTabs = () => {
...
+    const { t } = useTranslation();
     return (
         <>
             <StyledTabsTitlesContainer>
                 {settingsTabsData.map(tab => {
                     const { type, text, icon } = tab;
                     return (
                         <TabItemTitle
-                            title={text}
+                            title={t(`stetting.${text}.title`)}
```
Comment (lawvs) [inline libs/components/layout/src/settings-sidebar/ContainerTabs/ContainerTabs.tsx:79]: Please settings i18n at `settingsTabsData`

File: libs/components/layout/src/settings-sidebar/Settings/SettingsList.tsx [hunk 6]
```diff
@@ -32,7 +45,24 @@ export const SettingsList = () => {
 
                 return (
                     <ListItem key={item.name} onClick={() => item.onClick()}>
-                        {item.name}
+                        {t(`stetting.Settings.${item.name}`)}
```
Comment (lawvs) [inline libs/components/layout/src/settings-sidebar/Settings/SettingsList.tsx:48]: Please i18n at `useSettings`

File: libs/components/layout/src/settings-sidebar/Settings/SettingsList.tsx [hunk 7]
```diff
@@ -1,8 +1,21 @@
...
+    Switch,
+} from '@toeverything/components/ui';
+import { useTranslation } from 'react-i18next';
+import { options } from '../../i18n';
 import { useSettings } from './use-settings';
 
 export const SettingsList = () => {
     const settings = useSettings();
+    const { t, i18n } = useTranslation();
+    const changeLanguage = (event: any) => {
```
Comment (lawvs) [inline libs/components/layout/src/settings-sidebar/Settings/SettingsList.tsx:16]: Please explicit types

File: libs/components/layout/src/i18n/index.ts [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+    resources,
+    interpolation: {
+        escapeValue: false, // not needed for react as it escapes by default
+    },
+});
+
+export const options = [
+    { value: 'en', text: 'English' },
+    { value: 'zh', text: '简体中文' },
+];
```
Comment (lawvs) [inline libs/components/layout/src/i18n/index.ts:23]:
```suggestion
] as const;
```

**Comment by JimmFly** [comment]
update completed

File: libs/components/layout/src/settings-sidebar/Settings/SettingsList.tsx [hunk 9]
```diff
@@ -31,8 +44,25 @@ export const SettingsList = () => {
                 }
 
                 return (
-                    <ListItem key={item.name} onClick={() => item.onClick()}>
+                    <ListItem key={item.key} onClick={() => item.onClick()}>
                         {item.name}
+                        {item.key === 'Language' ? (
+                            <div style={{ marginLeft: '12em' }}>
+                                <Select
+                                    defaultValue="en"
```
Comment (lawvs) [inline libs/components/layout/src/settings-sidebar/Settings/SettingsList.tsx:52]:
```suggestion
                                    defaultValue={opetion[0].value}
```

**Review by lawvs — APPROVED** [review]

**Comment by lawvs** [comment]
Please update lock file by `pnpm install`

**Pull request merged**